### DISCUSSION TITLE
Creating moving background on Bio page

### DISCUSSION THREAD

**Discussion by G-gnome** [george-gca/george-gca.github.io#48] [General]
<img width="324" alt="Screenshot 2024-06-09 at 2 18 31 PM" src="https://github.com/george-gca/george-gca.github.io/assets/121409271/968975e6-e492-4cd5-af5d-2c288dbea811">

Hi, I was wondering how you were able to add this cool moving background pattern to your site. How did you do this? Thanks for your help.

**Comment by george-gca** [comment]
I got the code from [here](https://codepen.io/alexandrix/pen/oQOvYp), then did a lot of hacking into it to be able to make it behave like I intended it to.

**Reply by G-gnome** [reply]
Thanks man! I'll give it a shot
